xml/System.IO/FileStreamOptions.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Defines a variety of configuration options for <see. cref="T:System.IO.FileStream"/>.</summary>
```

Comment:
@adamsitnik @Jozkee does this description sound good to you? The class was not documented in triple slash.

Reply:
Sound good to me 👍